global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "version": "2.1.802",
+    "rollForward": "feature"
```

Comment:
Developers usually already have the .NET Core SDK installed, usually a newer version than required. `"rollForward"` works with the .NET Core 3.0 SDK and means that `2.1.802` becomes the minimum version instead of requiring an exact version. `"feature"` means that any `2.1.*` SDK is accepted.

(See links in https://github.com/dotnet/docs/issues/14480)

If a contributor does not have a 2.1.* SDK installed that is >= 2.1.802, they will get a message like this (I changed the versions for demonstration purposes):

```cs
A compatible installed .NET Core SDK for global.json version [2.1.802] from [C:\Users\Joseph\Source\Repos\nunit3-vs-adapter\global.json] was not found
Install the [2.1.802] .NET Core SDK or update [C:\Users\Joseph\Source\Repos\nunit3-vs-adapter\global.json] with an installed .NET Core SDK:
  2.1.801 [C:\Program Files\dotnet\sdk]
```